docs/reference/modules/gateway.asciidoc

```diff
@@ -48,3 +48,12 @@ as long as the following conditions are met:
     Recover as long as this many data nodes have joined the cluster.
 
 NOTE: These settings only take effect on a full cluster restart.
+
+==== Dangling indices
+
+When a node joins the cluster, any shards/indices stored in its local `data/` 
+directory which do not already exist in the cluster will be imported into the 
+cluster by default. This functionality is intended as a best effort to help 
```

Comment:
I think `by default` should be removed, because I don't think this is configurable (as it was, I believe, in 1.x).

Reply:
Thanks! Will fix.